packaging/debs/Debian/debian/rabbitmq-server.service

```diff
@@ -11,29 +10,14 @@ Group=rabbitmq
 UMask=0027
 NotifyAccess=all
 TimeoutStartSec=3600
-
```

Comment:
Please do not remove commented-out lines.

Reply:
Done